### DISCUSSION TITLE
[2933620] Call of Duty®: Black Ops 6

### DISCUSSION THREAD

**Discussion by plxt79** [pjy612/SteamManifestCache#909] [Question / Answer]
please provide keys and codes of call of duty black ops 6

**Comment by pjy612** [comment]
Why should I provide? You can buy the game yourself and extract the content you want

**Reply by john-xina-hub** [reply]
HOW

**Reply by pjy612** [reply]
google or wiki

**Reply by john-xina-hub** [reply]
We need vinoland to come back

**Reply by pjy612** [reply]
What is vinoland? Does it have anything to do with me?
